index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav class="container">
+            <h2 class="barItem">Welcome</h2>
+            <img class="barItem" src="images/Icon - Menu.png" alt="Burger Menu Icon">
+        </nav>
+    </header>
+    <section class="tittleContainer">
+        <h1 class="blue">Hey there. I'm Victor<br><span class="green"> I'm a software developer</span></h1>
+    </section>
+    <section class="helperText">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </section>
+    <section class="mediaContainer">
+        <a href="https://twitter.com/Victor_GonMa" target="_blank" ><img class="mediaIcon" src="images\twitter.png" alt="Twitter"></a>
+        <a href="https://www.linkedin.com/in/victor-manuel-gonzalez-massimi-a77265124/" target="_blank" ><img class="mediaIcon" src="images\Linkedin icon.png" alt="Twitter"></a>
+        <a href="https://github.com/Vgonma" target="_blank" ><img class="mediaIcon" src="images\medium.png" alt="Twitter"></a>
+        <a href="https://github.com/Vgonma" target="_blank" ><img class="mediaIcon" src="images\github.png" alt="Twitter"></a>
+        <a href="https://github.com/Vgonma" target="_blank" ><img class="mediaIcon" src="images\angellist.png" alt="Twitter"></a>
+    </section>
```

Comment:
- [ ] Excellent job adding the icons for your social media 💯 , The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`. Kindly, use `<ul>` and `<li>` tags for each `<a>` tag. 😄